doc-content/enterprise-only/installation/eap-users-create-proc.adoc

```diff
@@ -63,7 +75,32 @@ endif::[]
 If you installed {CENTRAL} and {KIE_SERVER} in the same server instance, you can create a single user that has both of these roles:
```

Comment:
Make this an optional step, not a note.
Optional: If you installed....
Then make this a note:
NOTE: You should install BC and KIE Server on different servers...

Reply:
Look at how Optional is used in other parts of the the doc. Not a label heading but an optional step.
. Optional: if you installed.